src/test/java/de/tum/in/www1/artemis/util/RequestUtilService.java

```diff
@@ -56,6 +56,14 @@ public RequestUtilService(MockMvc mvc, ObjectMapper mapper) {
         return new URI(res.getResponse().getHeader("location"));
     }
 
+    public <T> URI postWithContentType(String path, T body, HttpStatus expectedStatus, MediaType contentType) throws Exception {
```

Comment:
In my opinion you can also call this method `post` and use method overloading. In this way previous `post` method can call this `post` method with `MediaType.APPLICATION_JSON` content type